src/sinks/clickhouse.rs

```diff
@@ -385,6 +373,7 @@ compression = "none"
 [batch]
   max_size = 1
 [encoding]
+  format = "default"
```

Comment:
@binarylogic How do you feel about this? Because `EncodingConfig<E>` doesn't depend on `E: Default` we can't presume that there is a default `format` if there is some other fields in this table.

We could require the format if there are other fields present, or we could set a default for each sink and require `E: Default` on `EncodingConfig<E>`.